### PR TITLE
Add Choreo to PathWeaver replacements

### PR DESCRIPTION
**Pull request by spacey-sooty**

### CODE REVIEW

**Comment by calcmogul** [comment]
I'd link to both PathPlanner and Choreo, since Choreo can't do on-the-fly planning, and Choreo and PathPlanner have interop.

**Comment by TheTripleV** [comment]
No library wars pls

Just include both?

**Comment by spacey-sooty** [comment]
> No library wars pls

That wasn't the intention, Choreo is bundled in the installer whereas pathplanner isnt so imo it made more sense to recommend it. 

I can link to both

**Comment by spacey-sooty** [comment]
> It still reads as pathplanner being an afterthought/lesser alternative

IMO since choreo is in the installer we should first recommend that then suggest pathplanner if people need features it offers

File: source/docs/software/pathplanning/pathweaver/index.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # PathWeaver
 
-.. note:: Users may find a community driven project [PathPlanner](https://github.com/mjansen4857/pathplanner) as potentially more useful. PathPlanner improves upon traditional pathplanning applications with an intuitive user interface and swerve path following support. Note that WPILib offers no support for community projects.
+.. note:: Users may find [Choreo](https://sleipnirgroup.github.io/Choreo/) or [PathPlanner](https://github.com/mjansen4857/pathplanner) more usefu. They both have an inuitive user interface and swerve support. Users should already have access to Choreo as it is shipped with WPILib.
```
Comment (calcmogul) [inline source/docs/software/pathplanning/pathweaver/index.rst:3]:
```suggestion
.. note:: Users may find [Choreo](https://sleipnirgroup.github.io/Choreo/) or [PathPlanner](https://github.com/mjansen4857/pathplanner) more useful. They both have an intuitive user interface and swerve support. Users should already have access to Choreo as it is shipped with WPILib.
```

File: source/docs/software/pathplanning/pathweaver/introduction.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Introduction to PathWeaver
 
-.. note:: Users may find a community driven project [PathPlanner](https://github.com/mjansen4857/pathplanner) as potentially more useful. PathPlanner improves upon traditional pathplanning applications with an intuitive user interface and swerve path following support. Note that WPILib offers no support for community projects.
+.. note:: Users may find [Choreo](https://sleipnirgroup.github.io/Choreo/) or [PathPlanner](https://github.com/mjansen4857/pathplanner) more usefu. They both have an inuitive user interface and swerve support. Users should already have access to Choreo as it is shipped with WPILib.
```
Comment (calcmogul) [inline source/docs/software/pathplanning/pathweaver/introduction.rst:3]:
```suggestion
.. note:: Users may find [Choreo](https://sleipnirgroup.github.io/Choreo/) or [PathPlanner](https://github.com/mjansen4857/pathplanner) more useful. They both have an intuitive user interface and swerve support. Users should already have access to Choreo as it is shipped with WPILib.
```

**Review by calcmogul — CHANGES_REQUESTED** [review]

File: source/docs/software/pathplanning/pathweaver/introduction.rst (hunk 2 above)
Comment (jasondaming) [inline source/docs/software/pathplanning/pathweaver/introduction.rst:3]:
```suggestion
.. note:: Users may find [Choreo](https://sleipnirgroup.github.io/Choreo/) or [PathPlanner](https://github.com/mjansen4857/pathplanner) more useful. They both have an inuitive user interface and swerve support. Users should already have access to Choreo as it is shipped with WPILib.
```

**Review by jasondaming — APPROVED** [review]

**Pull request merged**